api/bases/watcher.openstack.org_watchers.yaml

```diff
@@ -39,6 +39,16 @@ spec:
           spec:
             description: WatcherSpec defines the desired state of Watcher
             properties:
+              apiContainerImageURL:
+                description: APIContainerImageURL
+                type: string
+              applierContainerImageURL:
+                description: DecisionEngineContainerImageURL
+                type: string
+              containerImage:
+                description: The service specific Container Image URL (will be set
+                  to environmental default if empty)
+                type: string
```

Comment:
Does the outer/umbrella `Watcher` CRD have an image itself?  Isn't it only the sub-CR that actually have images?

Reply:
Nope, Watcher CRD does not have image. Nice catch, let me remove that.

Reply:
Done!